extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/results/ExportResultsFrameProcessorFactory.java

```diff
@@ -88,7 +93,7 @@ public ExportStorageProvider getExportStorageProvider()
   }
 
   @Override
-  public ProcessorsAndChannels<Object, Long> makeProcessors(
+  public ProcessorsAndChannels<Object, Object> makeProcessors(
```

Comment:
Why not 
```suggestion
  public ProcessorsAndChannels<Object, List<String>> makeProcessors(
```

Reply:
Changing the type here would result in the arguments in thecall to mergeResults also having List<String>, which would cause a cast exception to be thrown instead of a nicer one, which we would like to avoid, right?